src/app/about/page.tsx

```diff
@@ -0,0 +1,142 @@
+import Head from "next/head";
+import Image from "next/image";
+import { headers } from "next/headers";
+import Init_Team from "./Init_Team";
+import Chapter_card from "./Chapter_card";
+import { Chapter } from "../api/chapters/route";
+import { Member } from "../api/eboard/route";
+
+export default async function About() {
+  const origin = headers().get("x-origin");
+  const chaptersRequest = await fetch(`${origin}/api/chapters`);
+  const chapters: Chapter[] = await chaptersRequest.json();
+  const eboardMembersRequest = await fetch(`${origin}/api/eboard`);
+  const team_members: Member[] = await eboardMembersRequest.json();
+  console.log(team_members);
+  return (
+    <>
+      <section className="mx-auto max-w-screen-lg ">
+        <Head>
```

Comment:
You should not use the head component in app directory: [next documentation](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#metadata)